File: I_godina/II_semestar/Objektno_orijentisano_programiranje_I/Vezbe/Zadaci/src/v02/Zadatak3.java
package v02;

public class Zadatak3 {

	public static void main(String[] args) {
		int n = 10;
		int arr[] = new int[n];
		arr[0] = 1;
		for(int i = 1; i < n; i++) {
			arr[i] = arr[i-1] + 10;
		}
		System.out.println("Niz je: ");
		for (int item : arr) {
			System.out.printf("%d ", item);
		}
		System.out.println();
	}

}

File: II_godina/III_semestar/Objektno_orijentisano_programiranje_2/Vezbe/vezba02/Vezbe/zadatak3.cpp
#include "zadatak3.h"
#include "input_helper.h"
#include <iostream>



void zadatak3() {
	int n = readInt("Unesite broj elemenata niza: ", true, false);
	int* arr = readIntArray(n);
	std::cout << "Unet je niz:\n";
	for (int i = 0; i < n; ++i) {
		std::cout << "arr[" << i << "] = " << arr[i] << std::endl;
	}
	int num[2] = { 0, 0 };
	for (int i = 0; i < n; i++) {
		++num[arr[i] % 2];
	}
	std::cout << "U nizu se nalazi " << num[0] << "parnih i " << num[1] << " neparnih elemenata\n";
	delete[] arr;
}